.github/workflows/update.yaml

```diff
@@ -0,0 +1,47 @@
+name: Update
+
+# FIXME: Delete when the PR is ready
+on:
+  push:
+    branches: 
+      - "gh-actions-update"
+
+# Enable for daily auto-updates
+# on:
+#     schedule:
+#     - cron: 07 8 * * * # 08:07 UTC is 00:07 PST
+
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I’d rename this to `ZONEDBOT_GITHUB_TOKEN` because vars with `GITHUB_` are semi-private.

Reply:
Done.